rules/xcodeproj.bzl

```
@@ -683,8 +683,11 @@ def _xcodeproj_impl(ctx):
         if _is_current_project_file(f)
     ]
 
+    attributes = {}
+    attributes.update(ctx.attr.project_attributes_overrides)
     xcodeproj_info = struct(
         name = paths.split_extension(project_name)[0],
+        attributes = attributes,
```

do you mean this?
```suggestion
        attributes = ctx.attr.project_attributes_overrides,
```
If so what i have in mind is that the patterm has been have a placeholder at line 686 so we can throw in any default attribute. Also it matches with how things declared for `options` and `settings`.

---

since ctx.attr.project_attributes_overrides has a default of `{}`, doing this assignment directly has no harm as well, I will do this after your reply.